README.md

```diff
@@ -109,6 +110,23 @@ or
 node index.js examples/bubble_sort.js examples/selection_sort.js
 ```
 
+### TOML Config
+
+You can create a [TOML](https://toml.io/en/) `.explainer-config.toml` config file that contains all of your options for the tool and place it in your `home` directory, i.e. (or simply modify and copy the one that is in the `examples directory`):
+```
```

Comment:
> MD040/fenced-code-language: Fenced code blocks should have a language specified markdownlint [MD040](https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md040.md)
> MD031/blanks-around-fences: Fenced code blocks should be surrounded by blank lines markdownlint [MD031](https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md031.md)
```suggestion

```toml
```

Reply:
NO WAY THERE IS TOML SUPPORT!!!! updated